src/kbmod/fake_data_creator.py

```diff
@@ -17,7 +17,7 @@
 from kbmod.work_unit import WorkUnit
 
 
-def add_fake_object(img, x, y, flux, psf=None):
+def add_fake_object(img, y, x, flux, psf=None):
```

Comment:
Mathematics uses `i` and `j` to denote `A[i, j]` - the `[row, column]` element. The row being "up" and "down" corresponds to `y` and column being "left" and "right" to `x`. Numpy uses `array[y, x]`, eigen uses `Matrix[y, x]` for both row and column major layouts, even `panda` `at`, `loc`, `iloc`, `iat`  methods use `[row, col]`. 
[Many of their methods ](https://numpy.org/doc/stable/reference/generated/numpy.meshgrid.html)in the documentation make sure to denote this appropriately and make a clear distinction between indexing in `xy` (i.e. Cartesian) notation vs `ij` (i.e. `yx`) notation and, if you're lucky enough to find an instance of where they actually use `y` and `x` instead of `[i, j]` or `[row, col]` in the documentation, you'll notice they denote access with `[y, x]`. This is common across the general user population as well. So I don't exactly agree that `(x, y)` is the common way of indexing arrays. 

We used `[x, y]` but marked `x` as the row. I guess it wasn't a big deal when access was done as `x*width + y` except it's confusing? In that case `x` is row, distinctly separated from it's Cartesian counterpart - the abscissa. Same for `y`. This becomes unbearable if we are clearly distinguishing an `Index` and an `Point` attributes of which imply they represent Cartesian coordinates and not an inverted naming convention of an array indexing scheme. 

So, I wouldn't necessarily agree that `(x, y)` is the standard convention in mathematics, or that it's something accepted by the general population. In fact I'd point that convention out as the less-normalized one.

Reply:
I'm a bit confused by the second to last paragraph. I've never seen mathematics represent cartesian coordinates as anything but (x, y). For the majority of high school math courses that is exactly how data is plotted. It's also the standard in all of the graphics and plotting libraries that I use (including matplotlib, tkinter, matlab, R).

I agree that matrix notation is commonly (row, col) and we should have the storage be consistent, but I would say that is distinct from how users (at least myself) would think about specifying the location within the image plane.

Reply:
If I wanted to add a value to the cartesian coordinate `(1, 2)` in numpy I would say `array[2, 1] += 1`. I wouldn't think of saying `array[x, y] = 10`?

If you're implying that we are distinguishing cartesian coordinates from indices this strongly then we should make the `get_pixel`, `set_pixel` `pixel_has_data` and so on also strongly `double` or `float`. This is not an obvious intention of the code.

Reply:
As discussed at the meeting, let's leave this as [y, x] when talking about accessing pixels.